File: lib/models/Otp.js
import mongoose from "mongoose";
const OtpSchema = new mongoose.Schema({
    email: {
        type: String,
    },
    otp: {
        type: String,
        unique: true
    },
    expiresAt: {
        type: Date
    }
})

const Otp = mongoose.models.Otp || mongoose.model("Otp", OtpSchema);
export default Otp;

File: backend/models/Otp.py
from mongoengine import Document, StringField, DateTimeField
from datetime import datetime, timedelta

class Otp(Document):
    email = StringField(required=True)
    otp = StringField(unique=True, required=True)
    expiresAt = DateTimeField(required=True)

    meta = {"collection": "otps"}  # explicitly set collection name
